android/src/main/java/com/svtek/reactnative/RNAndroidSharerModule.java

```diff
@@ -222,15 +281,14 @@ public void shareViaEmail(String filePath, String shareUrl, final Promise promis
       Intent shareIntent = new Intent();
       shareIntent.setAction(Intent.ACTION_SEND);
       shareIntent.setType("message/rfc822");
-      List<ResolveInfo> resolveInfoList =
-              this.reactContext.getPackageManager().queryIntentActivities(shareIntent, 0);
+      List<ResolveInfo> resolveInfoList = this.reactContext.getPackageManager().queryIntentActivities(shareIntent,
+              0);
```

Comment:
Please do not include changes that are not related to this commit's topic (`fix: fb share method`)